pages/Electric-Machines.md

```diff
@@ -0,0 +1,63 @@
+The Energy and Electricity category adds various machines which generate, consume or transfer energy (measured in Joules or J). It also includes [programmable androids](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Androids), which can be programmed to execute a series of tasks.
+
+Besides a way to generate electricity for your machines, an energy network requires an [Energy Regulator](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Energy-Regulator): without it, power transfer cannot happen.<br>
+You may also want to use [capacitors](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Energy-Capacitors), as they can store excess energy and extend the range of your network.
+
+Some machines have a buffer, meaning they can store a certain amount of energy. Unlike capacitors, however, if the energy isn't used before it reaches maximum capacity, it will be reset entirely.
+
+Electric machines are best used with a [Cargo Management System](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Cargo-Management) for automatic item transportation.
+
+For more information on the individual items in this category, see the table of contents below.
+
+## Table of contents
+### Energy management
+* [Energy Regulator](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Energy-Regulator)
+* [Energy Capacitors](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Energy-Capacitors)
+* [Multimeter](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Multimeter)
+* [Reactor Access Port](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Reactor-Access-Port)
```

Comment:
The Access Port is not really used to "manage" energy, it's an accessory to the Reactor.
I don't think it should fall under this category though.

Reply:
Okay, I've moved the Access Port to the "Energy generation" category, below reactors. It's not a generator by itself but hopefully that should be clear enough now that it's grouped with reactors